### PR TITLE
Points: signing up

### PR DESCRIPTION
**Pull request by benisgold**
Fixes APP-####

## What changed (plus any additional context for devs)
Christian built out the following flows for signing up for the points program. I hooked it up to backend logic and tied it all together. At this point, the v1 feature is complete aside from referral links (+ deeplinking), twitter sharing, and some ui updates.

Screenshot 2023-11-30 at 4 38 30 PM
Screenshot 2023-11-30 at 4 38 52 PM

### CODE REVIEW

File: src/screens/points/content/ClaimContent.tsx [hunk 1]
```diff
@@ -70,6 +71,7 @@ export default function ClaimContent() {
               alignItems: 'center',
               justifyContent: 'center',
             }}
+            onPress={() => navigate(Routes.CONSOLE_SHEET)}
```
Comment (skylarbarrera) [inline src/screens/points/content/ClaimContent.tsx:74]: dont love the name of this sheet fwiw

**Review by skylarbarrera — APPROVED** [review]
mainly i18n tings

File: src/screens/points/ConsoleSheet.tsx [hunk 2]
```diff
@@ -0,0 +1,1086 @@
...
+          />
+        </Animated.View>
+      </Box>
+      <AnimatePresence
+        condition={showSignInButton && !didConfirmOwnership}
+        duration={300}
+      >
+        <NeonButton label="􀎽 Sign In" onPress={signIn} />
+      </AnimatePresence>
+      <AnimatePresence condition={showSwapOrBuyButton} duration={300}>
```
[the remark this answers is not in the document]
Comment (benisgold) [reply src/screens/points/ConsoleSheet.tsx:222]: yes, for people w/ no retroactive points

File: src/screens/points/ConsoleSheet.tsx [hunk 3]
```diff
@@ -0,0 +1,1086 @@
...
+              backgroundColor: globalColors.white50,
+            }}
+            top={{ custom: 6 }}
+            width={{ custom: 36 }}
+          />
+          <ClaimRetroactivePointsFlow
+            pointsProfile={pointsProfile}
+            didConfirmOwnership={didConfirmOwnership}
+            setShowSignInButton={setShowSignInButton}
+            setShowSwapOrBuyButton={setShowSwapOrBuyButton}
```
[the remark this answers is not in the document]
Comment (benisgold) [reply src/screens/points/ConsoleSheet.tsx:212]: for people who don't have retroactive points. it'll tell them to swap or buy eth depending on their eth balance

**Pull request merged**